AppManager/app_manager_server.py

```diff
@@ -475,9 +475,12 @@ def stop_app(version_key):
   for entry in version_entries:
     yield unmonitor_and_terminate(entry)
 
-  if not remove_logrotate(project_id):
-    logging.error("Error while setting up log rotation for application: {}".
-      format(project_id))
+  try:
+    projects_manager[project_id]
```

Comment:
Minor point, but `if project_id not in projects_manager` might be easier to read since we're not really trying to perform an operation here.